content/departments/bizops/tools/amplitude.md

```diff
@@ -9,97 +9,36 @@
-
-If you're adding a new event property, please add it to the [data map](https://docs.google.com/spreadsheets/d/1wz958I67BKWWY0jKY3oXKhlrGZ9ucKmv0CM94K-5NVs/edit#gid=408201559).
-
-```
-JSON_EXTRACT(argument,
-  '$.code_search.results.results_count') AS results_count,
-JSON_EXTRACT(argument,
-  '$.code_search.results.alert') AS alert,
-CASE
-  WHEN name = 'ViewRepogroup:python' THEN 'python'
-  WHEN name = 'ViewRepogroup:golang' THEN 'golang'
-  WHEN name = 'ViewRepogroup:android' THEN 'android'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:stanford' THEN 'stanford'
-  WHEN name = 'ViewRepogroup:cncf' THEN 'cncf'
-ELSE
-NULL
-END
-repogroup_name,
-```
-
-### Adding user properties
-
-User properties are the attributes of individual users. The `amplitude_user_characteristics` table defines all user properties, and we join in this table with every event that we pass to Amplitude. These user properties include every A/B test from `ab_test_users`. See more in the A/B testing doc (coming soon).
-
-If an additional user property is added to this table, it will only be applied to events and users going forward. User properties are sent to Amplitude along with events, so new properties will only be added when a user triggers an event that is sent to Amplitude. When adding a user property, please add it to the [data map](https://docs.google.com/spreadsheets/d/1wz958I67BKWWY0jKY3oXKhlrGZ9ucKmv0CM94K-5NVs/edit#gid=735397811)
+1. Events are stored in the [dotcom_events.events_usage](<[url](https://console.cloud.google.com/bigquery?project=telligentsourcegraph&_ga=2.7211002.912372869.1639779794-1385560724.1639779794&pli=1&ws=!1m5!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3sevents_usage)>)
+2. Incremental events are formatted to fit Amplitude's schema, enhanced with user properties (company name, company region, etc) in [this scheduled query](<[url](https://console.cloud.google.com/bigquery/scheduled-queries/locations/us/configs/6319611b-0000-203e-8729-94eb2c0924fc/runs?project=telligentsourcegraph)>), and a JSON is created in a [GCS bucket](<[url](https://console.cloud.google.com/storage/browser/event-usage-to-amplitude/data/2022/08;tab=objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&project=telligentsourcegraph&prefix=&forceOnObjectsSortingFiltering=false)>).
+3. Every 5 minutes Amplitude checks for new data in the GCS bucket to ingest.
```

Comment:
Right now the cadence is set to one hour, can we keep it to one hour? 
The reason being that when a folder gets created in GCS we organize by month/day/hour and on our scheduled query we have `overwrite = True` so `export001.json` would get overwritten every 5 mins by our scheduled query but amplitude checks for "new" files and `export001.json` wouldnt be considered a new file.

If we do want 5 minutes lets change our `overwrite` settings to `false`?
<img width="380" alt="image" src="https://user-images.githubusercontent.com/32119652/187721685-390d5a3f-e1e3-470a-bd4a-9a5f410aa176.png">